spec/stub_spec.lua

```diff
@@ -98,4 +98,13 @@ describe("Tests dealing with stubs", function()
     assert.is.equal("foo", test.key())
   end)
 
+  it("returns multiple given values", function()
+    stub(test, "key", "foo", "bar")
+
+    arg1, arg2 = test.key()
+
+    assert.is.equal("foo", arg1)
+    assert.is.equal("bar", arg2)
+  end)
+
```

Comment:
I'd guess the issue is that `unpack` only works on the sequence part of the table, i.e. `unpack({"foo", nil, "bar"})` could only return `"foo"`. To fix it, you could save number of arguments in an upvalue and then pass it to `unpack`, e.g. `local return_values_number = select("#", ...)` and then `return unpack(return_values, 1, return_values_number)`.